cfgov/v1/atomic_elements/organisms.py

```diff
@@ -774,15 +773,17 @@ class Media:
 class AudioPlayer(blocks.StructBlock):
     heading = v1_blocks.HeadingBlock(required=False)
     body = blocks.RichTextBlock(required=False)
-    audio_file = AbstractMediaChooserBlock(
+    audio_file = blocks.CharBlock(
```

Comment:
A more appropriate block would be Wagtail's built-in [URLBlock](https://docs.wagtail.org/en/stable/reference/streamfield/blocks.html#wagtail.blocks.URLBlock), since we only need to validate that the provided user value is a valid URL.

Reply:
If we're just rendering the URL's into `audio` blocks (like you've done in audio-player.html, `URLBlock` is the correct thing to use. It's better than a CharBlock in that it actually requires the url to be valid.

The Hyperlink atom is generally what we use when rendering links in some pre-defined structure and comes with its own template, so it actually *wouldn't* be appropriate here, since you're just popping a URL into separate template structure.